src/app/static/src/tests/testHelpers.ts

```diff
@@ -111,14 +117,51 @@ export const expectChangeLanguageMutations = (commit: Mock) => {
     });
 };
 
-export const expectErrorReportOpen = (wrapper: DOMWrapper<any>, row = 0) => {
-    const link = wrapper.findAll(".dropdown-item").at(row);
-    link?.trigger("click");
-
+export const expectErrorReportOpen = async (wrapper: VueWrapper<any>, row = 0) => {
+    const link = wrapper.findAll(".dropdown-item")[row];
+    await link.trigger("click");
     expect(wrapper.findComponent(ErrorReport).props("open")).toBe(true);
 }
 
 export function expectArraysEqual(result: any[], expected: any[]) {
     expect(result).toEqual(expect.arrayContaining(expected));
     expect(expected).toEqual(expect.arrayContaining(result));
 }
+
+/*
+This are functions that let us easily put in the translate directive into mount and shallowMount test
+functions. There are ts-ignores as shallowMount/mount require a DefineComponent type as their first
+argument however if we replace the C generic type below to "C extends DefineComponent" then we get
+typescript errors when we use shallowMountWithTranslate with our normal vue components.
+
+Since this is just for testing, ts-ignore was used for this special case.
+*/
+// @ts-ignore
```

Comment:
Could you not ignore at all ?

Reply:
the comment above this line was trying to say that if I remove the ignore then mount has type constraints for the component argument (it needs to extend type `DefineComponent`) however if I add that constraint to the generic type `C` then when I use the `mountWithTranslate` function, typescript complains about the type of our Vue components, since this is testing, I thought the easiest way is just to ts-ignore this and we just have to make sure that we put in vue components into the mount functions